src/index.tsx

```diff
@@ -157,7 +173,7 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = default
     }
 
     return () => {
-      if (status === SpeechRecognitionStatus.STARTED) {
+      if (status === SpeechRecognitionStatus.STARTED && !options.continuous) {
```

Comment:
This was preventing the `continuous` option for actually continue listening after the first result